### PR TITLE
Cut v0.2.9 release

### PR DESCRIPTION
**Pull request by ellistarn**
**Issue, if available:**

**Description of changes:**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,7 +17,7 @@ dev: verify test ## Run all steps in the developer loop
 
 ci: verify licenses battletest ## Run all steps used by continuous integration
 
-release: publish helm docs ## Run all steps in release workflow
+release: codegen publish helm docs ## Run all steps in release workflow
```
Comment (akestner) [inline Makefile:20]: What changed to necessitate including this here?

File: pkg/utils/parallel/workqueue.go [hunk 2]
```diff
@@ -1,3 +1,17 @@
+/*
```
Comment (akestner) [inline pkg/utils/parallel/workqueue.go:1]: Are these license headers manually added?

**Review by akestner — APPROVED** [review]
Couple quick questions, but nothing that would require any changes

File: Makefile (hunk 1 above)
Comment (ellistarn) [reply Makefile:20]: I realized it was missing. It's just an extra safe check that all of our files have the correct headers, that apidocs are up to date, etc.

File: pkg/utils/parallel/workqueue.go (hunk 2 above)
Comment (ellistarn) [reply pkg/utils/parallel/workqueue.go:1]: Added with `make codegen`

**Pull request merged**